spec/queue.js

```diff
@@ -0,0 +1,76 @@
+import chai, { expect, assert } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+  'use strict'
+  let myQueue
+  beforeEach( () => {
+    myQueue = new Queue()
+  })
+
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('#enqueue(element)', () => {
+    it('should add an element to the back of the queue', () => {
+      expect(() => myQueue.enqueue("foo")).to.alter(() => myQueue.length(), { from: 0, to: 1 })
```

Comment:
Consider adding some whitespace to make this more readable - I have a hard time processing the embedded functions with parameters with it condensed on one line like this.